sql/dd/impl/types/object_table_impl.cc

```diff
@@ -35,7 +35,9 @@ Object_table_impl::Object_table_impl()
       m_target_def(),
       m_actual_present(false),
       m_actual_def(),
-      m_hidden(true) {
+      m_hidden(true),
+      was_target_encryption_set{false},
```

Comment:
Better names are m_target_encryption_set and m_actual_encryption_set

Why do you always use "was_"?